.github/workflows/dotnet.yml

```
@@ -2,18 +2,20 @@ name: Backend CI
 
 on:
   push:
-    branches: [ main, backend-dev, release ]
+    branches: [ main, backend-dev-#46, release ]
```

This needs to be changed when merged again :) 

```suggestion
    branches: [ main, backend-dev, release ]
```

---

Yes. That might fail again so I redirected it to the dev-#46 branch... during the sprint release when the changes are merged we can update that...